Documentation/doc/CMakeLists.txt

```diff
@@ -185,7 +185,7 @@ function(configure_doxygen_package CGAL_PACKAGE_NAME)
 endfunction()
 
 # set up the directories and variables
-set(CGAL_DOC_MATHJAX_LOCATION "https://cdn.mathjax.org/mathjax/latest"
+set(CGAL_DOC_MATHJAX_LOCATION "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.2"
```

Comment:
Do we really want to set that variable? Recent versions of Doxygen have a default for `MATHJAX_RELPATH` that is already right. And we can trust the Doxygen author better than us for upgrading that version when new versions of MathJax are published.

Reply:
AFAIS, this is only use in resource files for doxygen 1.8.4, so this should be fine.

Reply:
If I grep the variable name, I see:
```
doc/resources/1.8.13/BaseDoxyfile.in
1574:MATHJAX_RELPATH        = ${CGAL_DOC_MATHJAX_LOCATION}
```
Is not that Doxygen configuration variable used?

Reply:
You're right, I fixed that in #2760 in 48658fe5f6 (since new resources files were added for 1.8.14).